spack/scripts/local-build.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+
+set -e
+
+if [[ ! -z $SPACK_COMMAND ]]; then
+  echo "..."
+  echo
+else
```

Comment:
This whole `SPACK_COMMAND` thing I will revert if you still want this script in the repo. I only used this approach during failed experiments doing finch builds on macOS.